zulip/integrations/bridge_with_matrix/README.md

```diff
@@ -27,20 +26,37 @@ details mentioned below. For example:
 * If you are running from the Zulip GitHub repo: `python matrix_bridge.py --write-sample-config matrix_bridge.conf`
 
 ### 1. Zulip endpoint
-1. Create a generic Zulip bot, with a full name like `IRC Bot` or `Matrix Bot`.
-2. Subscribe the bot user to the stream you'd like to bridge your IRC or Matrix
-   channel into.
+1. Create a generic Zulip bot, with a full name such as `Matrix Bot`.
+2. The bot is able to subscribe to the necessary streams itself if they are
+   public. (It may even create them if they do not exist and if the bot user
+   has enough rights to create streams.)
+   Otherwise, you need to add the bot manually.
 3. In the `zulip` section of the configuration file, enter the bot's `zuliprc`
    details (`email`, `api_key`, and `site`).
 4. In the same section, also enter the Zulip `stream` and `topic`.
 
 ### 2. Matrix endpoint
-1. Create a user on [matrix.org](https://matrix.org/), preferably with
-   a formal name like to `zulip-bot`.
+1. Create a user on the matrix server of your choice, e.g. [matrix.org](https://matrix.org/),
+   preferably with a formal name like to `zulip-bot`.
```

Comment:
Typo spotted in the earlier version of this README: `s/like to/such as/`.
Though I don't understand the "formal name" requirement? I think @neiljp wrote this documentation.

Reply:
thanks! I think Neil meant "descriptive name" :)